README.md

```diff
@@ -65,7 +65,7 @@ Defaults to `false`. New in 0.11.0.
 * `timestampFormat`: Specifies an additional timestamp format that will be tried when parsing values as `TimestampType` 
 columns. The format is specified as described in [DateTimeFormatter](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html).
 Defaults to try several formats, including [ISO_INSTANT](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_INSTANT),
-including variations with offset timezones or no timezone (defaults to UTC). New in 0.12.0.  
+including variations with offset timezones or no timezone (defaults to UTC). If a custom format is used, it will check if it has a timezone in the pattern. If not, it will uses the default spark timezone from `spark.sql.session.timeZone`. New in 0.12.0.  
```

Comment:
Let's say "As of 0.16.0, if a custom format pattern is used without a timezone, the default Spark timezone specified by `spark.sql.session.timeZone` will be used" and similarly below